calendar/stream_implementer.bal

```diff
@@ -68,11 +73,16 @@ class CalendarStream {
     private final http:Client httpClient;
     private CalendarListOptional? optional;
     private string? pageToken;
+    private ClientOAuth2ExtensionGrantHandler clientHandler;
+    private final string? userAccount;
 
-    isolated function init(http:Client httpClient, CalendarListOptional? optional = ()) {
+    isolated function init(http:Client httpClient, ClientOAuth2ExtensionGrantHandler clientHandler, 
+                            CalendarListOptional? optional = (), string? userAccount = ()) {
```

Comment:
`CalendarListOptional ` is not a good name for an input variable. Is it `calendarsToAccess`?

Reply:
'CalendarListOptional' contains only optional values. 'calendarsToAccess' name may confuse users as required variables. Shall we use use different name?

Reply:
What kind of optional values it is having? Optionality of a field is denoted by 

* `?` in coding
* In low code they will not get rendered by default. 

In that way users know it is optional. Do we need to name input variable name to denote it optional?

Reply:
https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/174c8083d5b4df7553766ff5d09cc610b0ac1456/calendar/records.bal#L515

Valid point. We can change the name to `calendarsToAccess.` So shall we use this name for annotation as well?

Reply:
Yes please. `Calendars to Access`

Reply:
Fixed in ad948e118e8946134a67fa31a86f882aca532620